docs/source/international_languages.md

```diff
@@ -17,10 +17,10 @@ In this topic, we're going to cover:
 Hyperledger Fabric documentation is being translated into many different
 languages. For example:
 
-* [Chinese](https://github.com/hyperledger/fabric-docs-i18n/tree/master/docs/locale/zh_CN)
-* [Malayalam](https://github.com/hyperledger/fabric-docs-i18n/tree/master/docs/locale/ml_IN)
-* [Brazilian Portuguese](https://github.com/hyperledger/fabric-docs-i18n/tree/master/docs/locale/pt_BR)
-* [Japanese](https://github.com/hyperledger/fabric-docs-i18n/tree/master/docs/locale/ja_JP)
+* [Chinese](https://github.com/hyperledger/fabric-docs-i18n/tree/main/docs/locale/zh_CN)
+* [Malayalam](https://github.com/hyperledger/fabric-docs-i18n/tree/main/docs/locale/ml_IN)
+* [Brazilian Portuguese](https://github.com/hyperledger/fabric-docs-i18n/tree/main/docs/locale/pt_BR)
+* [Japanese](https://github.com/hyperledger/fabric-docs-i18n/tree/main/docs/locale/ja_JP)
```

Comment:
Do you want to keep this as main. The docs translations are actually happening on the `release-2.2` branch over there, so the `master` branch and future `main` don't contain the vast majority of the translations, and dont contain any new translations for these languages that have been done in the last 8 months. Perhaps we should just make is `release-2.2` as that is the default branch in this repo

Reply:
good point, i think that is worth doing, but let's merge this first.